airsonic-main/src/main/java/org/airsonic/player/domain/Album.java

```diff
@@ -143,11 +152,15 @@ public void setGenre(String genre) {
     }
 
     public int getPlayCount() {
-        return playCount;
+        return playCount.get();
     }
 
     public void setPlayCount(int playCount) {
-        this.playCount = playCount;
+        this.playCount.set(playCount);
+    }
+    
+    public void incrementPlayCount() {
+        this.playCount.incrementAndGet();
```

Comment:
Added for primary use case of the previously used setter.
For instance, `setPlayCount(getPlayCount() + 1)` is not atomic. Two threads may access getPlayCount() at the same time and increment by 1 each.

This allows us to update atomically without read-eval-write cycle which might not work with concurrent threads.